src/main/java/frc/robot/robot_manager/RobotManager.java

```diff
@@ -250,6 +283,11 @@ public void unjamRequest() {
   public void intakeRequest() {
     switch (getState()) {
       case CLIMBING_1_LINEUP, CLIMBING_2_HANGING -> {}
+      case IDLE_WITH_GP -> {
+        if (!queuer.hasNote()) {
+          setStateFromRequest(RobotState.INTAKING);
+        }
+      }
```

Comment:
i would prefer to not have this logic. it seems pretty harmless to enter the intake state and then just exit it right after. makes it more deterministic as to what happens if i click the intake button as well (intake button always means `INTAKING`)